### ISSUE TITLE
iOS 15 version

### ISSUE THREAD

**Issue by nunofgs** [patrickshox/Keys#30]
Hi @patrickshox, thanks for the great extension!

Any plans to port it to iOS 15?

**Comment by patrickshox** [comment]
Yes, to iPadOS.

**Comment by kevcube** [comment]
@patrickshox is this still on the roadmap?

**Comment by nunofgs** [comment]
@kevcube I just bought it on the App Store, happily. I simply followed the app store link in the readme, from my device: https://apps.apple.com/us/app/keys-for-safari/id1494642810?mt=12

**Comment by kevcube** [comment]
> @kevcube I just bought it on the App Store, happily. I simply followed the app store link in the readme, from my device: https://apps.apple.com/us/app/keys-for-safari/id1494642810?mt=12

Yes thank you! That's the same way I found it on my iPad. I'm happy to have it, but still confused why I couldn't get to it from search.

**Comment by bbb651** [comment]
Would be nice to also have iOS (iPhone) support as well, obviously less useful than iPadOS but you can connect a keyboard as well, so if it’s just checking a checkbox I don’t see a reason not to allow it. Game controller support can potentially be useful as well for devices like Backbone One but that’s more work to support for a very niche use case.

**Comment by kevcube** [comment]
> Would be nice to also have iOS (iPhone) support as well, obviously less useful than iPadOS but you can connect a keyboard as well, so if it’s just checking a checkbox I don’t see a reason not to allow it. Game controller support can potentially be useful as well for devices like Backbone One but that’s more work to support for a very niche use case.

+1 also visionOS
